README.md

```diff
@@ -4,32 +4,27 @@ It features two processes that take turns calling functions `f()` and `g()` *in
 
 ![Screenshot](https://user-images.githubusercontent.com/5462697/41799127-e976f7a6-7678-11e8-8b06-fb6b4ee1d005.png)
 
-How?
+How it Works
 ===
-There’s nothing new in Lua being flexible: it allows to override all the memory management by passing a custom allocator to [`lua_newstate`](https://www.lua.org/manual/5.3/manual.html#lua_newstate).
```

Comment:
The main reasons for all the changes I have made is to make it easier for a reader to understand. It could be grammatically correct, but may still be confusing to read.

1. Yes, you are correct in using colon to connect two independent clauses together. I changed the colon to a period as they imply the same meaning for me when I had read the sentence for the first time. So using colon or period are both okay, but I chose periods for the sake of avoiding confusion between the two clauses.

2. "the" is to be used for definite articles and when a noun is specified. Using "all the memory management" is not specific as "all" was used, so "the" was removed. Source: https://owl.english.purdue.edu/owl/resource/540/1/

> Definite Article: the
> The definite article is used before singular and plural nouns when the noun is specific or particular. The signals that the noun is definite, that it refers to a particular member of a group.